src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -1497,6 +1500,20 @@ public async Task RotateApiKeyAsync(Organization organization)
             await ReplaceAndUpdateCache(organization);
         }
 
+        public async Task DeleteSsoUserAsync(Guid userId, Guid? organizationId)
+        {
+            await _ssoUserRepository.DeleteAsync(userId, organizationId);
+            if (organizationId.HasValue)
+            {
+                var organizationUsers = await _organizationUserRepository.GetManyByUserAsync(userId);
+                var organizationUser = organizationUsers.FirstOrDefault(orgUser => orgUser.OrganizationId == organizationId.Value);
```

Comment:
There is a `Task<OrganizationUser> GetByOrganizationAsync(Guid organizationId, Guid userId)`.

Reply:
Fixed